docker/docker-compose.yml

```diff
@@ -1,5 +1,12 @@
 version: "3.3"
 
+x-logging:
```

Comment:
More important would be to add these to the [lemmy-ansible docker-compose.yml](https://github.com/LemmyNet/lemmy-ansible/blob/main/templates/docker-compose.yml)

This docker-compose.yml should be for test setups anyway, where log size doesn't really matter. Next release, we'll probably only reference that ansible one.

Reply:
Its still good to have.